config/packages/easy_admin/participation.yaml

```diff
@@ -7,6 +7,7 @@ easy_admin:
                 list: 'easy_admin/Participation/list.html.twig'
             list:
                 filters: ['conference', 'participant']
+                item_permission: ['SHOW_PARTICIPATION']
```

Comment:
You tried to be creative on this once :) 
But as I said in the previous comment, we prefer `PARTICIPATION_SHOW`. It namespace better attributes